workflows/amplicon-mgnify/mgnify-amplicon-pipeline-v5-complete/mgnify-amplicon-pipeline-v5-complete.ga

```diff
@@ -0,0 +1,13844 @@
+            "tool_version": null,
+            "type": "data_input",
+            "uuid": "5bcd8781-4c1c-43a7-98d3-35e8ff87e1b4",
+            "when": null,
+            "workflow_outputs": []
+        },
+        "2": {
+            "annotation": "Covariance models.",
+            "content_id": null,
+            "errors": null,
+            "id": 2,
+            "input_connections": {},
+            "inputs": [
+                {
+                    "description": "Covariance models.",
+                    "name": "Covariance models"
+                }
+            ],
+            "label": "Covariance models",
+            "name": "Input dataset",
+            "outputs": [],
+            "position": {
+                "left": 2.8348931366990726,
+                "top": 177.84517063139768
+            },
+            "tool_id": null,
+            "tool_state": "{\"optional\": false, \"tag\": null}",
+            "tool_version": null,
+            "type": "data_input",
+            "uuid": "dca1f52f-e90f-40cc-a3da-0f523ff2e4e7",
+            "when": null,
+            "workflow_outputs": []
+        },
+        "3": {
+            "annotation": "Average quality required. (Single-end)",
+            "content_id": null,
+            "errors": null,
+            "id": 3,
+            "input_connections": {},
+            "inputs": [
+                {
+                    "description": "Average quality required. (Single-end)",
+                    "name": "SE Trimmomatic - SLIDING WINDOW - Average quality required"
+                }
+            ],
+            "label": "SE Trimmomatic - SLIDING WINDOW - Average quality required",
+            "name": "Input parameter",
+            "outputs": [],
+            "position": {
+                "left": 42.60539394991072,
+                "top": 438.7423844130483
+            },
+            "tool_id": null,
+            "tool_state": "{\"default\": 15, \"parameter_type\": \"integer\", \"optional\": true}",
+            "tool_version": null,
+            "type": "parameter_input",
+            "uuid": "5d990811-1c1b-4de5-ae02-a9560e49c0a1",
+            "when": null,
+            "workflow_outputs": [
+                {
+                    "label": null,
+                    "output_name": "output",
+                    "uuid": "9d18d09f-451d-46b5-84c2-4d167067365e"
+                }
+            ]
+        },
+        "4": {
+            "annotation": "Cut bases off the start of a read, if below a threshold quality. (Single-end)",
+            "content_id": null,
+            "errors": null,
+            "id": 4,
+            "input_connections": {},
+            "inputs": [
+                {
+                    "description": "Cut bases off the start of a read, if below a threshold quality. (Single-end)",
+                    "name": "SE Trimmomatic - LEADING"
+                }
+            ],
+            "label": "SE Trimmomatic - LEADING",
```

Comment:
In the case of the tool itself you are right, it is optional. Here it is actually set to a default value, because this is the pipeline default value..